docs/tutorials/AddRemark.md

```diff
@@ -5,7 +5,7 @@ title: "Tutorial: Adding a command"
 
```

Comment:
I think AddRemark.md can be removed since it is essentially just the Remark command step-by-step tutorial.

Reply:
Maybe we could just leave it be for now, because this can make it easier for someone to trace through our code in our context. If we had to refer to the old `AddRemark.md` where the method and file names still used `AddressBook` and `Person`, it could be harder to trace. I just feel it might still be useful in the future when we need to check if we did something correctly or wrongly.

Reply:
Sure no problem.